### ISSUE TITLE
How to get a 3d object from a new video?

### ISSUE THREAD

**Issue by adrida** [zju3dv/4K4D#22]
Hello, thank you for the great work. I was wondering if it was possible with the current implementation to pass as input 2/3 videos from a new scene and get a 3d animation file that could be rendered using Blender for example?

**Comment by dendenxu** [comment]
Hi @adrida thanks for the interest.
Currently, 4K4D focuses more on the novel view synthesis side and uses a different rendering pipeline than that of the traditional triangle meshes. 
Thus for creating 3D assets directly (with such a small view count), you should consider looking into other 3D/4D neural reconstruction methods that focus on surface quality and use human priors (with SDF field or directly optimizes meshes), like [AniSDF](https://github.com/zju3dv/animatable_nerf) or [Relightable Avatar](https://zju3dv.github.io/relightable_avatar/)

**Comment by adrida** [comment]
I see, thanks a lot for your answer and for the references I will check them out. Any idea on projects that would export 3d animated assets of large scenes with multiple persons?
I could have more video angles as input if needed and I don't necessarily need to be able to edit the 3d asset. I am looking to be able just "replay" the scene from different angles and navigate in the 3d space (a bit like static nerf approaches where you can export the 3d reconstruction of an appartement in VR and be in it).

**Comment by dendenxu** [comment]
> Any idea on projects that would export 3d animated assets of large scenes with multiple persons?

Ah, for multi-person reconstruction, I recommend checking out [CloseMocap](https://github.com/zju3dv/CloseMoCap) and [MultiNB](https://github.com/zju3dv/EasyMocap?tab=readme-ov-file#news) (the first news of EasyMocap).

> I could have more video angles as input if needed and I don't necessarily need to be able to edit the 3d asset. I am looking to be able just "replay" the scene from different angles and navigate in the 3d space (a bit like static nerf approaches where you can export the 3d reconstruction of an appartement in VR and be in it).

For replaying the reconstruction, there are generally two approaches:
1. Support PCVR in the specific rendering algorithm (like a connector with Unity VR or just natively supports communication with VR devices). An example is the VR-NeRF method you mentioned. We're planning on adding this functionality to [EasyVolcap](https://github.com/zju3dv/EasyVolcap), on which 4K4D is built.
2. Export the reconstructed sequence to a renderable format and replay it (like exporting a mesh sequence). This might come at some significant quality loss.

**Comment by dendenxu** [comment]
Indeed, the field is advancing fast and I also think that future is not far away.

> Any way I could help for the VR feature you are planning on adding to EasyVolcap?

I couldn't think of any specific problem to solve as of right now, but we are always looking forward to a PR of any kind. Feel free to contribute!
